src/lib/commandBarConfigs/applicationCommandConfig.ts

```diff
@@ -79,5 +85,195 @@ export function createApplicationCommands({
     },
   }
 
-  return isDesktop() ? [textToCADCommand] : [textToCADCommand]
+  const addKCLFileToProject: Command = {
+    name: 'add-kcl-file-to-project',
+    displayName: 'Add file to project',
+    description: 'Add KCL file or 3D model to new or existing project.',
+    needsReview: true,
```

Comment:
```suggestion
    needsReview: false,
```

In using this it was really only the review step that I think could confuse users with the naming issue we talked about today. Users have to read to hit submit on the review step, and then they'll see the command is called "Add file to project" and not "Create project from sample" or whatever. Just a thought, but I think we have other ways to help that like we discussed so nbd.

Reply:
I can swap this to false. You are correct, adding stuff won't ever overwrite data so it shouldn't need review.